app/models/__init__.py

```diff
@@ -175,6 +175,7 @@
 SYSTEM_MAP_SIZE_KEY = "system_map_size"
 TEST_CASE_ID_KEY = "test_case_id"
 TEST_CASE_PATH_KEY = "test_case_path"
+TEST_CASE_LOG_LINE_KEY = "test_case_log_line"
```

Comment:
Please rename to `LOG_LINE_NUMBER_KEY`, as log lines are actual lines from the log.  The new field you're adding is intended to be a line number.  Also I don't think we need to call it `TEST_CASE_LOG_LINE_NUMBER_KEY` since this could be used in theory in other places where a log line number is needed.